META.tyxml.template

```diff
@@ -0,0 +1,2 @@
+# JBUILDER_GEN
+archive(byte,toploop) += "tyxml_top.cma"
```

Comment:
Wouldn't it be better to use `requires` and `tyxml.top` as opposed to `archive` here? It's a bit more general.

Reply:
The goal of this is to auto-load `tyxml.top` in the toplevel when `tyxml` is #required. Do you have another way of doing that?

Reply:
`requires(byte, toploop) += "tyxml.top"` should do the same thing but also handle dependencies if your toplevel has dependencies. Anyway, I'm not too sure what's best here. None of it is gonna work with `$ jbuilder utop` anyway :/

Reply:
Ah, I see. Yes, your version is better. I should probably open a ticket about better support for this.